engine.py

```diff
@@ -504,9 +504,48 @@ def save_to_path(self, document, path):
 
             # make the document being processed the active document
             self.adobe.app.activeDocument = document
-
-            # TODO need to use appropriate save options here (jpg, tiff, etc)
-            document.saveAs(self.adobe.File(path))
+        
+            (_, ext) = os.path.splitext(path)
+            ext = ext.lower()
+
+            # these extensions are supported by photoshop
+            # PICTResourceSaveOptions is skipped for now, need a way to differentiate PICT
```

Comment:
Now that there are conditionals for each type, I'd suggest putting the comments inside the appropriate condition.